js/db.js

```diff
@@ -756,6 +756,93 @@ define(["dao", "globals", "lib/bluebird", "lib/davis", "lib/underscore", "util/h
                     playerFeatStore.createIndex("tid", "tid", {unique: false});
                 }());
             }
+            if(event.oldVersion <= 13) {
```

Comment:
Oops, sorry-- just saw this, I don't get e-mail notifications from Github. Will make the needed changes and update again